src/shogun/machine/LinearMachine.cpp

```diff
@@ -35,11 +35,11 @@ void CLinearMachine::init()
 	bias = 0;
 	features = NULL;
 
-	SG_ADD(&m_w, "w", "Parameter vector w.", MS_NOT_AVAILABLE);
-	SG_ADD(&bias, "bias", "Bias b.", MS_NOT_AVAILABLE);
+	SG_ADD(&m_w, "w", "Parameter vector w.", ParameterProperties());
```

Comment:
things like this would be MODEL

Reply:
Both the weight and bias right?

Reply:
yes. We dont need to fix all the wrongly added parameter in this PR though, just put a few examples in